addons/website/static/src/components/translator/translator.js

```diff
@@ -72,6 +73,26 @@ export class AttributeTranslateDialog extends Component {
                 }
                 this.formEl.el.append(divEl);
             }
+            // When the user can edit the href, we automatically propose to
+            // change the text content.
+            if (this.translation.href) {
+                const linkTextEl = this.props.node.querySelector("[data-oe-model='ir.ui.view']");
+                if (linkTextEl) {
+                    const divEl = document.createElement("div");
+                    divEl.classList.add("mb-3");
+                    const labelEl = document.createElement("label");
+                    labelEl.classList.add("form-label");
+                    labelEl.textContent = _t("Link label");
+                    const inputEl = document.createElement("input");
+                    inputEl.classList.add("form-control");
+                    inputEl.value = linkTextEl.textContent;
+                    inputEl.addEventListener("change", () => {
+                        linkTextEl.textContent = inputEl.value;
```

Comment:
If I translate the telephone number in the navbar, I lost the phone icon :phone:
Is it possible to fix that ?
Before:
![image](https://github.com/odoo/odoo/assets/52911687/ceb0ee84-05be-449b-b5ce-055ab4593969)
After:
![image](https://github.com/odoo/odoo/assets/52911687/031ce716-c29f-48be-b6b0-f731b7693a3a)